.github/workflows/pr-golangci-lint.yaml

```diff
@@ -1,4 +1,4 @@
-name: golangci-lint
+name: PR golangci-lint
 
 on:
```

Comment:
Sorry one additional note. I just merged the release documentation. Can you please update release-tasks.md to list all relevant files we have to update when we add a new branch (Create a release branch > 7.)